use-cases/GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,276 @@
+     crw-rw-rw- 1 root root  10, 196 May 18 15:37 vfio
+   ```
+
+5. Start a Kata container with GPU device:
+
+   ```
+   $ sudo docker run -it --runtime=kata-runtime --rm --device /dev/vfio/1 -v /dev:/dev debian /bin/bash
+   ```
+
+   Running `lspci` within the container should show the GPU device among the PCI devices.
+
+   ```
+   $ lspci -nn -D
+     0000:00:00.0 Class [0600]: Device [8086:1237] (rev 02)
+     0000:00:01.0 Class [0601]: Device [8086:7000]
+     0000:00:01.1 Class [0101]: Device [8086:7010]
+     0000:00:01.3 Class [0680]: Device [8086:7113] (rev 03)
+     0000:00:02.0 Class [0604]: Device [1b36:0001]
+     0000:00:03.0 Class [0780]: Device [1af4:1003]
+     0000:00:04.0 Class [0100]: Device [1af4:1004]
+     0000:00:05.0 Class [0002]: Device [1af4:1009]
+     0000:00:06.0 Class [0200]: Device [1af4:1000]
+     0000:00:0f.0 Class [0300]: Device [8086:1616] (rev 09)
+   ```
+
+   Additionally, you should be able to access the device node for the graphics device:
+
+   ```
+   $ ls /dev/dri
+     card0  renderD128
+   ```
+
+## GVT-g with Kata Containers
+
+For GVT-g, you need to append `i915.enable_gvt=1` in addition to `intel_iommu=on`
+to your host kernel command line. Make sure to reboot your host on
+making these changes.
+
+Follow  the steps below to pass an Intel Graphics device in GVT-d mode to a Kata Container:
+
+1. Find the BDF for GPU device:
+
+   ```
+   $ sudo lspci -nn -D | grep Graphics
+     0000:00:02.0 VGA compatible controller [0300]: Intel Corporation Broadwell-U Integrated Graphics [8086:1616] (rev 09)
+   ```
+
+   Run the above command to find out the BDF for the GPU device on host.
+   From the above output, PCI addres "0000:00:02.0" is assigned to the GPU device.
+   This BDF will be used later to unbind the GPU device from host.
+
+2. Choose the MDEV (Mediated Device) type for VGPU(Virtual GPU):
+
+   For background on mdev types, please follow this [kernel documentation](https://github.com/torvalds/linux/blob/master/Documentation/vfio-mediated-device.txt)
+
+   * List out the mdev types for the VGPU:
+
+     ```
+     $ BDF="0000:00:02.0"
+
+     $ ls /sys/devices/pci0000:00/$BDF/mdev_supported_types
+       i915-GVTg_V4_1  i915-GVTg_V4_2  i915-GVTg_V4_4  i915-GVTg_V4_8
+     ```
+
+   * Inspect the mdev types and choose one that fits your requirement:
+
+     ```
+     $ cd /sys/devices/pci0000:00/0000:00:02.0/mdev_supported_types/i915-GVTg_V4_8 && ls
+       available_instances  create  description  device_api  devices
+
+     $ cat description
+       low_gm_size: 64MB
+       high_gm_size: 384MB
+       fence: 4
+       resolution: 1024x768
+       weight: 2
+
+     $ cat available_instances
+       7
+     ```
+
+     The output of file `description` represents the GPU resources that will be assigned to the VGPU with specified MDEV type.<br/>
+     The output of file `available_instances` represents the remaining amount of VGPU can be created with specified MDEV type.
```

Comment:
The output of file `available_instances` represents the remaining amount of VGPU you can create with specified MDEV type.